cinema/serializers.py

```diff
@@ -1,6 +1,11 @@
 from rest_framework import serializers
 
-from cinema.models import Movie
+from cinema.models import (
+    Movie,
+    Genre,
+    Actor,
+    CinemaHall
+)
 
 
 class MovieSerializer(serializers.Serializer):
```

Comment:
Consider using `serializers.ModelSerializer` for the `MovieSerializer` class. This would automatically generate the `create` and `update` methods based on the model fields, reducing boilerplate code.